docs/features/contributor-insights.md

```diff
@@ -17,22 +17,62 @@ Monitor individual commit histories, track who is actively engaged, identify alu
 
-There are three ways to add a list:
+
+## Contributor Insight for Contributors
+
+
+Contributor insight offer contributors a way to gain insight into the world of open source contributions, including:
+
+- **Track personal contributions**: If you create a contibutor insight of your personal contributions, you can track them and see how you're doing over time, as well as get a breakdown of the type of contributions you're making, including commits, created PRs, reviewed PRs, and created issues. 
+
+- **Learn about what other contributors are doing in open source**: You can learn where other contributors are contributing, what they're working on, and how active they are in the open source community.
+
+- **See who's using OpenSauced Highlights feature**: You can see who's using the Highlights feature and show them support by giving an emoji reaction.
+
+- **Track contributors for job opportunities**: If you're looking for a job, you can track the activity of the contributors you're interested in working with. This helps you understand the activity level of the repositories and the types of contributions being made, and you can even identify which company you want to contribute to.
+
+There are Three ways to add contributors to track:
+
+1. **Search for Contributors**: Search for ontributors  from repositories in your workspace.
+2. **Import Contribuors**: Paste a list of contributors to track on yor workspace.
+3. **Sync GitHub Following**: Import users that follow from contributors.
+
+Once your contributor insight is created, you'll be able to see an overview of all the contributors, a graph of their activity, and a detailed breakdown of the contributors.
+
+## How to Create a Contributor Insight Page
+
+After logging in, navigate to your OpenSauced workspace. Click on the navigation bar of your workspace home page. Click on the plus sign beside insight. Two options will display, which are ; New repository insight and New contributor insight. Click on the New contributor insight. Then add the contributors by searching for their names or import contributors by syncing your GitHub following.
+
+ [Contributor Insights Page](https://app.opensauced.pizza/hub/lists/).
+If you're having trouble syncing your Organization or Team, check out our [Sync Your GitHub Team in our FAQs](../welcome/faqs.md#sync-your-github-team).
 
 1. **Explore Contributors**: Use our explore tool to find contributors and create your list.
 2. **Sync your GitHub Team**: Connect to your GitHub to create a Contributor Insight Page from a team in your organization.
 3. **Import your GitHub Following**: Connect to your GitHub to create a Contributor Insight Page with all the contributors you follow.
 
 Once your Contributor Insight Page is created, you'll be able to see an overview of all the contributors in your list, a graph of their activity, and a detailed breakdown of the contributors.
 
-:::tip
```

Comment:
Why are you removing the tip here?

Reply:
> Why are you removing the tip here?

I did not see a tip in the modified file. I saw the tip while I was trying to fix the merge conflict. A reason I removed it.

Reply:
Ah, I like the tip because it makes it stand out.